modules/bitgo/test/v2/unit/tss/ecdsa.ts

```diff
@@ -0,0 +1,242 @@
+import {
+  readSignedMessage,
+  Ecdsa,
+  ECDSA,
+  ECDSAMethods,
+} from '@bitgo/sdk-core';
+import * as openpgp from 'openpgp';
+import * as should from 'should';
+
+type KeyShare = ECDSA.KeyShare;
+const encryptNShare = ECDSAMethods.encryptNShare;
+const createCombinedKey = ECDSAMethods.createCombinedKey;
+
+describe('test tss helper functions', function () {
+  let mpc: Ecdsa;
+
+  let userKeyShare: KeyShare;
+  let backupKeyShare: KeyShare;
+  let bitgoKeyShare: KeyShare;
+
+  let userKey;
+  let backupKey;
+  let bitgoKey;
+
+  let userGpgKeypair: { publicKey: string, privateKey: string };
```

Comment:
Not sure whether this type is defined else where should be right where we can re use this?

Reply:
I don't see one in opengpg package. I'll define one in test for reuse